### DISCUSSION TITLE
Which regular expressions are used?

### DISCUSSION THREAD

**Discussion by muznyo** [ImranR98/Obtainium#521] [General]
I'd like to know them all

**Comment by ImranR98** [comment]
Not sure what you mean. The regex filters are user-defined and would depend on their specific needs.

**Reply by muznyo** [reply]
I want to exclude a specific word for example

**Reply by ImranR98** [reply]
I think you're looking for something like `^(word)` but it really depends on the context. There are a lot of resources online to learn regex. You can use regex101.com to play around and test things.

**Reply by muznyo** [reply]
> I think you're looking for something like `^(word)` but it really depends on the context. There are a lot of resources online to learn regex. You can use regex101.com to play around and test things.
> 

I added this [repo](https://github.com/moneytoo/Player)

there are 2 package versions, and I didn't want the one with "-legacy"

I was able to exclude it with this: `^((?!-legacy).)*$`
